airflow/providers/apache/spark/hooks/spark_submit.py

```diff
@@ -282,10 +313,57 @@ def _resolve_connection(self) -> dict[str, Any]:
     def get_conn(self) -> Any:
         pass
 
+    def _create_keytab_path_from_base64_keytab(self, base64_keytab: str, principal: str | None) -> str:
+        _uuid = uuid.uuid4()
+        temp_dir_path = Path(tempfile.gettempdir()).resolve()
+        temp_file_name = f"airflow_keytab-{principal or _uuid}"
+
+        keytab_path = temp_dir_path / temp_file_name
+        staging_path = temp_dir_path / f".{temp_file_name}.{_uuid}"
+
+        try:
+            keytab = base64.b64decode(base64_keytab)
+        except Exception as err:
```

Comment:
What with add to this exception?

Reply:
https://github.com/apache/airflow/pull/40757/commits/912cab06280d8b6fa42279764cc28d9a5bda79ca done!

Reply:
Great!